fabric/src/components/CurrencyInput/index.tsx

```diff
@@ -9,9 +9,12 @@ export type CurrencyInputProps = React.InputHTMLAttributes<HTMLInputElement> &
   Omit<InputBoxProps, 'inputElement' | 'rightElement'> & {
     currency?: string
     onSetMax?: () => void
+    variant?: 'small' | 'large'
+    handleChange?: (value: number) => void
```

Comment:
I think `onChange` is more standard, and in line with `onSetMax`?

```suggestion
    onChange?: (value: number) => void
```